monai/networks/blocks/activation.py

```diff
@@ -12,18 +12,30 @@
 import torch
 from torch import nn
 
-from monai.utils import optional_import
+from monai.utils import PT_BEFORE_1_7, PT_BEFORE_1_9
 
-if optional_import("torch.nn.functional", name="mish")[1]:
+if PT_BEFORE_1_9:
```

Comment:
Hi @yiheng-wang-nv, this version check is not robust because some nvidia pytorch images use a different versioning convention. for this PR I'd suggest `optional_import` to check the existence of `silu`

Reply:
Thanks @wyli ! I will modify that.